guidance/models/llama_cpp/_llama_cpp.py

```diff
@@ -60,18 +61,28 @@ def __init__(self, model_obj, chat_template=None):
         # get the bytes strings for all the tokens
         tokens = []
         for i in range(tokenizer.llama.n_vocab()):
-            tok = tokenizer.llama.detokenize([i])  # note that detokenize returns bytes directly
+            tok = tokenizer.llama.detokenize(
+                [i]
+            )  # note that detokenize returns bytes directly
             if tok == b"":
-                tok = llama_cpp.llama_token_get_text(model_obj.model, i)  # get text rep of special tokens
+                tok = llama_cpp.llama_token_get_text(
+                    model_obj.model, i
+                )  # get text rep of special tokens
```

Comment:
I'm really not a fan of this `black` artifact on trailing lines :\

Wonder if one of the `black` authors had a job where they were paid by number lines of code produced 😅. 
If we really do need a code formatter, I think we should prioritize investigations into alternatives.

Reply:
Hadn't noticed that I accidentally ran `black` on this file. Have fixed.